model/src/cfd.rs

```diff
@@ -885,6 +885,8 @@ impl Cfd {
             hours_to_charge as i64,
         )?;
 
+        tracing::debug!(%hours_to_charge, funding_fee = %funding_fee.fee, "Accepting rollover proposal");
```

Comment:
This will be a duplicate now with the other one. Shouldn't we drop the other one in favor of this?